definintion-of-iss-project.md

```diff
@@ -0,0 +1,58 @@
+# Title
+
+The Definition of an InnerSource Software (ISS) Project. We will largely take the ideas that follow
+from the definition of an Open Source Software (OSS) Project. Note, the definition of an OSS Project
+follows from [opensource.com's definition](https://opensource.com/resources/what-open-source). Further,
+[RedHat also supplies a defintion of OSS project](https://www.redhat.com/en/topics/open-source/what-is-open-source).
```

Comment:
As @MaineC already suggested I think using or at least adding the OSIs definitions here would have merit. 
https://opensource.org/osd